tests/handlers/botas/test_botas_shipped_advisory.py

```diff
@@ -207,3 +232,47 @@ def test_filter_bundles_by_pinned_related_images(self, service):
         temp_mock.get_build.assert_has_calls([call(bundle_list[0]),
                                              call(bundle_list[1])])
         self.assertEqual(nvrs, {bundle_list[0]})
+
+    def test_no_original_nvr_found(self):
```

Comment:
I don't understand why this test is named this way. Could you please explain?

Reply:
It's tests that `get_published_original_nvr` will return the first entry when there are no other events to test. That's why I thought about that name.

Reply:
How about renaming this to `test_get_published_original_nvr_single_event` or something similar?